XamlStyler.Console/XamlStyler.Console.csproj

```diff
@@ -1,109 +1,40 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProjectGuid>{5284FEA7-FC0B-45EA-8B16-9DA7F5E6D7B6}</ProjectGuid>
     <OutputType>Exe</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
```

Comment:
Can we target 3.1? I tried this out on my machine, and it seems to eliminate the need to install 2.2 run time. 3.1 is also an LTS version and contains some useful features for packaging + reducing binary size. https://devblogs.microsoft.com/dotnet/announcing-net-core-3-1/